barretenberg/cpp/pil/vm2/execution.pil

```diff
@@ -26,16 +26,38 @@ include "nullifier_check.pil";
 namespace execution;
 
 pol commit sel; // subtrace selector
+// Subtrace operation id
+pol commit subtrace_operation_id;
+
+// Subtrace Dispatch selectors
```

Comment:
Comment that we don't need to constrain them to be boolean because they come from a precomputed table? Same in others?